extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -304,6 +303,22 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
 
+  /**
+   * Listens to navigation events.
+   */
+  startListeningToNavigationEvents() {
```

Comment:
I'm ok with renaming these, but was there some reason this was pulled out?  Just want to make sure I'm not missing something

Reply:
This was needed for another improvement I tried, I reverted these changes, thanks for pointing it out!